src/app/teamSheets/_TeamSheetListItem/index.tsx

```diff
@@ -0,0 +1,56 @@
+'use client'
+
+import { usePathname } from 'next/navigation'
+
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome'
+import { faEdit, faShareFromSquare } from '@fortawesome/free-solid-svg-icons'
+
+import type { TeamSheetWithTeam } from '@types'
+
+interface TeamSheetListItemProps {
+  teamSheet: TeamSheetWithTeam,
+}
+
+const TeamSheetListItem = (props: TeamSheetListItemProps) => {
+  const { teamSheet } = props || {}
+  const { editId, shareId, team, title } = teamSheet
+  const { title: teamTitle } = team
+
+  const pathname = usePathname()
```

Comment:
pathname won't help us here because we are not adding to the `/teamsheets` route, so this can be removed and this can be a server component